### ISSUE TITLE
separate ticket to keep track of little styling things that don't affect interactivity and functionality so don't need immediate fixes but need to be done before deployment

### ISSUE THREAD

**Issue by kelsey-21** [nss-evening-cohort-10/nutshell-jurassic-world#77]
- style `vendors` cards to match `equipment` cards w/ margins, + box shadows
- add 10px top margin to staff cards
- center staff cards
- add padding to
